test/e2e/common/setup-kubestellar.sh

```diff
@@ -41,10 +41,53 @@ source "$COMMON_SRCS/setup-shell.sh"
 
 :
 : -------------------------------------------------------------------------
-: Create a Kind hosting cluster with nginx ingress controller and KubeFlex controller-manager
+: Create a hosting cluster with nginx ingress controller and KubeFlex controller-manager
 :
-kflex init --create-kind $disable_chatty_status
-: Kubeflex kind cluster created.
+if [ -z "$USE_K3D" ]; then
+   kflex init --create-kind $disable_chatty_status
+   kubectl config rename-context kind-kubeflex kubeflex
```

Comment:
maybe kf-mgt or kflexmgt or something that shows that this is the management cluster where KF is running.